### PR TITLE
Listen to Auth Events JTBD guide

### PR DESCRIPTION
**Pull request by heatheramz**
### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [X] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [X] JS
- [ ] iOS
- [ ] Android
- [ ] Flutter
- [X] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://link.com)` 
            HTML: `<a href="https://link.com">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: src/pages/lib/auth/auth-events/q/platform/[platform].mdx [hunk 1]
```diff
@@ -22,17 +22,298 @@ export const getStaticProps = (context) => {
...
+Before you begin, you will need:
+
+- An Amplify project with the Auth category configured
+- The Amplify libraries installed and configured
+
+## Expose hub events triggered in response to auth actions
+
+You can use Amplify Hub with its built in Amplify Auth events to subscribe a listener using a publish-subscribe pattern and capture events between different parts of your application. The Amplify Auth category publishes in the `auth` channel when auth events such as `signIn`, `signUp`, and `signOut` happen independent from your app code.
+
+<InlineFilter filters={["js"]}>
```
Comment (abdallahshaban557) [inline src/pages/lib/auth/auth-events/q/platform/[platform].mdx:38]: is there a reason there is an inline filter JUST for JS here?

Comment (abdallahshaban557) [reply src/pages/lib/auth/auth-events/q/platform/[platform].mdx:38]: the info between JS and RN seems repetitive to me

File: src/pages/lib/auth/auth-events/q/platform/[platform].mdx [hunk 2]
```diff
@@ -22,17 +22,298 @@ export const getStaticProps = (context) => {
...
-import ios1 from '/src/fragments/lib/auth/ios/hub_events/10_listen_events.mdx';
+## Next steps
+
+Now that you can listen to auth events you may also want to add some additional features. We recommend you learn more about:
+
+<InlineFilter filters={["js"]}>
+
+- [Manage user profile](/lib/auth/manageusers/q/platform/js/)
+- [Set up password change and recovery](/lib/auth/password_management/q/platform/js/)
+- [Use with Server-Side Rendering (SSR)](/lib/ssr/q/platform/js/)
```
Comment (abdallahshaban557) [inline src/pages/lib/auth/auth-events/q/platform/[platform].mdx:299]: let us remove the SSR page from here - and then combine the links for the inline filters for both JS + RN

**Review by abdallahshaban557 — COMMENTED** [review]
Left some comments

File: src/pages/lib/auth/auth-events/q/platform/[platform].mdx (hunk 1 above)
Comment (heatheramz) [reply src/pages/lib/auth/auth-events/q/platform/[platform].mdx:38]: @abdallahshaban557 yeah this is for the hyperlinks inline - once we have the new site we can do relative links but for now we have to list JS and RN links separate b/c of the switcher placement

File: src/pages/lib/auth/auth-events/q/platform/[platform].mdx (hunk 2 above)
Comment (heatheramz) [reply src/pages/lib/auth/auth-events/q/platform/[platform].mdx]: @abdallahshaban557 Removed SSR but the links are set up so they only see the links relevant to their platform. In the new site structure we can take these away but for now the platform switcher does not allow for relative links

**Review by abdallahshaban557 — APPROVED** [review]
LGTM

**Review by israx — APPROVED** [review]

**Review by timngyn — APPROVED** [review]

**Pull request merged**